### PR TITLE
UI changes in View Files fragment

### LINKED ISSUE #215: UI changes in View Files fragment
screen shot 2018-07-14 at 4 02 42 pm

We have a lot of options in the dropdown menu.

### What to do?

- Create a new strip at the top
- It should have name & icons for following : New directory, Move files, move files to home directory, delete directory.
- The functioning should be same as before
- Remove these options from drop down menu,

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #215

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ x] `./gradlew assembleDebug assembleRelease`
- [ x] `./gradlew checkstyle`

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
Add a screenshot

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 1]
```diff
@@ -132,6 +135,46 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         // Populate data into listView
         populatePdfList();
+        LinearLayout newdir = root.findViewById (R.id.layout_new);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: Change the name of ids to` new_dir`, `move_to_dir`, `move_to_home_dir`, `delete_dir`

**Comment by shiv07tiwari** [comment]
screenshot_2018-07-15-15-06-11-222_swati4star createpdf
screenshot_2018-07-15-15-06-19-895_swati4star createpdf

**Comment by shiv07tiwari** [comment]
Screenshots added @Swati4star

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 2]
```diff
@@ -132,6 +135,46 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
...
+        newdir.setOnClickListener(new View.OnClickListener() {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: Directly use butterknife @OnClick

File: app/src/main/res/menu/activity_view_files_actions.xml [hunk 3]
```diff
@@ -36,24 +36,4 @@
         android:title="@string/share"
         android:orderInCategory="4"
         app:showAsAction="never" />
-    <item
-        android:id="@+id/directory"
-        android:title="@string/new_dir"
```
Comment (Swati4star) [inline app/src/main/res/menu/activity_view_files_actions.xml]: I guess, you can now remove these strings from strings.xml file

**Comment by shiv07tiwari** [comment]
Made the changes @Swati4star

**Comment by Swati4star** [comment]
@shiv07tiwari new ui screenshot?

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 4]
```diff
@@ -238,7 +233,12 @@ public void showSnack(int resID) {
     /**
      * Moves selected files to home directory
      */
-    private void moveFilesToHomeDirectory() {
+    @OnClick(R.id.move_to_home_dir)
+    void moveFilesToHomeDirectory() {
+        if (!mViewFilesAdapter.areItemsSelected()) {
+            showSnack(R.string.snackbar_no_pdfs_selected);
+            return;
+        }
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: indentation error

File: app/src/main/res/drawable/ic_arrow_forward_black_24dp.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
```
Comment (Swati4star) [inline app/src/main/res/drawable/ic_arrow_forward_black_24dp.xml]: change the name of files also from black to white. It creates a new confusion later

**Comment by shiv07tiwari** [comment]
screenshot_2018-07-15-16-05-26-696_swati4star createpdf

@Swati4star  New UI

**Comment by Swati4star** [comment]
UI looks good now.

**Comment by shiv07tiwari** [comment]
@Swati4star  Made the changes.

**Pull request merged**